server/schedule/operator/step.go

```diff
@@ -29,20 +31,38 @@ import (
 // OpStep describes the basic scheduling steps that can not be subdivided.
 type OpStep interface {
 	fmt.Stringer
-	ConfVerChanged(region *core.RegionInfo) bool
+	ConfVerChanged(region *core.RegionInfo) uint64
```

Comment:
It seems this request both adjust the old function and also add the new implementation for `OpStep`.  I recommended that we can split this request into 2 steps:

1. adjust `ConfVerChanged1` and fix old function
2. introduce new implementtation for `OpStep`

Reply:
fine, this PR includes only the `uint64` part first